<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Laboratory</class>
 <widget class="QWidget" name="Laboratory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>320</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>320</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QToolBox" name="toolBox">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>291</width>
     <height>261</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>168</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Запрос 1</string>
    </attribute>
    <widget class="QLineEdit" name="quest1query">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Иванов Иван Иванович</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doIt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>221</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать анализы пациента</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>168</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Запрос 2</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать заявки из больницы №</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doIt2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>261</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="questquery2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>45</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>291</width>
      <height>168</height>
     </rect>
    </property>
    <attribute name="label">
     <string>Запрос 3</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Показать все завяки от врача</string>
     </property>
    </widget>
    <widget class="QPushButton" name="doit3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="quest1query_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Зайцев Никита Алексеевич</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
